src/communication/README.md

```diff
@@ -0,0 +1,56 @@
+# Azure CLI communication Extension #
+This is the extension for communication
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name communication
+```
+
+### Included Features ###
+#### communication communication-service ####
+##### Create #####
+```
+az communication communication-service create --name "MyCommunicationResource" --location "Global" \
```

Comment:
`az communication service` should be the right command group?

Reply:
Fixed